plugins/catalog-backend/src/ingestion/processors/AzureApiReaderProcessor.ts

```diff
@@ -119,17 +119,19 @@ export class AzureApiReaderProcessor implements LocationProcessor {
         userOrOrg,
         project,
         '_apis',
-        'sourceProviders',
-        'TfsGit',
-        'filecontents',
+        'git',
+        'repositories',
+        repoName,
+        'items',
       ].join('/');
 
-      url.search = [
-        `repository=${repoName}`,
-        `commitOrBranch=${ref}`,
-        `path=${path}`,
-        'api-version=6.0-preview.1',
-      ].join('&');
+      const queryParams = [`path=${path}`];
```

Comment:
Oh hey I just want to check. Since this is using raw string manipulation, does it handle url encoded characters correctly? Like if the path contains a file with spaces or similar.